.circleci/config.yml

```diff
@@ -75,6 +78,10 @@ jobs:
     - run: ./architect build
     - store_test_results:
         path: /tmp/results
+    - persist_to_workspace:
+        root: .
+        paths:
+        - ./app-operator
```

Comment:
Ah okay, This is why original build failing quite long time, https://quay.io/repository/giantswarm/app-operator